### PR TITLE
[JENKINS-48711] - Enable Multi-stage build for PCT

### PR DESCRIPTION
**Pull request by oleg-nenashev**
After that we can build PCT image on DockerHub: https://hub.docker.com/r/jenkins/pct/

@reviewbybees @raul-arabaolaza

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,33 +20,26 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
-FROM maven:3.5.2-jdk-8
+FROM maven:3.5.2-jdk-8 as builder
 LABEL Description="Base image for running Jenkins Plugin Compat Tester (PCT) against custom plugins and Jenkins cores" Vendor="Jenkins project"
```
Comment (raul-arabaolaza) [inline Dockerfile]: Maybe this label needs to be moved from the builder to the final distributed image?

**Comment by raul-arabaolaza** [comment]
:bee: Sorry for the delay, I needed to learn about multi-stage builds before.

File: Dockerfile (hunk 1 above)
Comment (oleg-nenashev) [reply Dockerfile]: yes, you are right

**Comment by oleg-nenashev** [comment]
@raul-arabaolaza fixed, thanks for catching it!

**Comment by raul-arabaolaza** [comment]
Re :bee: This time without any comment :)

**Pull request merged**